rust-toolchain.toml

```diff
@@ -1,2 +1,3 @@
 [toolchain]
 channel = "nightly-2025-02-16"
+components = ["rustc-dev"]
```

Comment:
I think this should include `rust-src` as well, as that is now a requirement to run `mir-json-translate-libs` (see #93).